app/models/concerns/form526_rapid_ready_for_decision_concern.rb

```diff
@@ -93,25 +92,10 @@ def single_issue?
     disabilities.size == 1
   end
```

Comment:
is `single_issue?` still needed?

Reply:
Removed as per our conversation in Slack.